examples/common/pigweed/rpc_services/Device.h

```diff
@@ -418,6 +419,22 @@ class Device : public pw_rpc::nanopb::Device::Service<Device>
         {
             mCommissionableDataProvider.SetSpake2pVerifier(ByteSpan(request.verifier.bytes, request.verifier.size));
         }
+
+        if (Server::GetInstance().GetCommissioningWindowManager().CommissioningWindowStatusForCluster() ==
+            app::Clusters::AdministratorCommissioning::CommissioningWindowStatus::kBasicWindowOpen)
```

Comment:
This is not quite right.  The `CommissioningWindowStatusForCluster` will say "not open" if we are in our on-boot commissioning window (which was not opened via the Administrator Commissioning cluster).  But in that state you do in fact want to reopen the commissioning window.

But if we _are_ in the "opened by cluster" state, then the reopening needs to persist the "who opened the commissioning window" state at the very least, to avoid breaking things.

Reply:
Done